database_mockup/src/animalkeeping/ui/PersonsView.java

```diff
@@ -68,6 +68,19 @@ public void initialize(URL location, ResourceBundle resources) {
         usernameLabel.setText("");
         userroleLabel.setText("");
         personsTable.getSelectionModel().getSelectedItems().addListener(new PersonTableListChangeListener());
+        idCol = new TableColumn<>("id");
+        idCol.setCellValueFactory(data -> new ReadOnlyLongWrapper(data.getValue().getId()));
+        subjectCol = new TableColumn<>("subject");
+        subjectCol.setCellValueFactory(data -> new ReadOnlyStringWrapper(data.getValue().getSubject().getName()));
+        typeCol = new TableColumn<>("treatment");
+        typeCol.setCellValueFactory(data -> new ReadOnlyStringWrapper(data.getValue().getTreatmentType().getName()));
+        startDateCol = new TableColumn<>("start");
+        startDateCol.setCellValueFactory(data -> new ReadOnlyObjectWrapper<>(data.getValue().getStart()));
+        endDateCol = new TableColumn<>("end");
+        endDateCol.setCellValueFactory(data -> new ReadOnlyObjectWrapper<>(data.getValue().getEnd()));
+        treatmentTable.getColumns().clear();
+        treatmentTable.getColumns().addAll(idCol, subjectCol, typeCol, startDateCol, endDateCol);
+
```

Comment:
These additions result from a wrong merge?! with pr #25 this table was not created here by hand but uses the TreatmentTypeTable class